tests/device_tests/ckb/spx_ref.py

```
@@ -0,0 +1,97 @@
+def _load() -> ctypes.CDLL:
+    global _lib
+    if _lib is not None:
+        return _lib
+    if not _REF.is_dir():
+        raise BuildUnavailable(f"{_REF} is missing (submodule not checked out?)")
+
+    out = Path(tempfile.mkdtemp(prefix="spx_ref_"))
+    (out / "stub.c").write_text(_STUB)
+    lib_path = out / "libspx.so"
+    try:
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)